package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "embedded_sagecell.js",
+  "version": "1.0.0",
+  "description": "Sage cell that can be embedded into a webpage",
+  "main": "main.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build:copystatic": "mkdir -p build/vendor && cp static/sagecell.css static/fontawesome.css static/colorpicker/js/colorpicker.js static/colorpicker/css/colorpicker.css build/vendor",
+    "build:deps": "npm run build:copystatic && node fetch_vendor_js.mjs",
+    "build": "webpack --mode production",
+    "watch": "webpack --watch --mode development"
+},
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sagemath/"
+  },
+  "keywords": [
+    "sage"
+  ],
+  "author": "Andrey Novoseltsev",
```

Comment:
That is highly inaccurate. At this point I think you should add yourself to the list, and we must also include Jason Grout and a few of his students who made massive work on the project. Probably we can use github contributors to judge who should be here.

Reply:
Tell me what to put and I'll change it.